export-pprof.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Generate a timestamp
+timestamp=$(date +%Y-%m-%d_%H-%M-%S)
+
+# Function to fetch goroutine profile
+fetch_goroutine_profile() {
```

Comment:
Is this script used anywhere? Couldn't find the references or docs.

Reply:
This script is used for manual profiling which we've added as part of this feature (disabled by default). I probably need to add some documenting on how to use it